03week/towersOfHanoi.js

```diff
@@ -17,27 +29,65 @@ function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
-}
-
-function movePiece() {
-  // Your code here
-
-}
-
-function isLegal() {
-  // Your code here
+};
 
-}
+const movePiece = (startStack, endStack) => {
+  const start = stacks[startStack];
+  const end = stacks[endStack];
+  end.push(start.pop());
+};
 
-function checkForWin() {
-  // Your code here
+const isLegal = (startStack, endStack) => {
+  const SSLastNum = stacks[startStack][stacks[startStack].length-1];
+ const ESLastnum = stacks[endStack][stacks[endStack].length-1];
+ if (ESLastnum > SSLastNum) {
+   return true;
+ } else {
+   return false;
+ };
+};
 
-}
+const checkForWin = () => {
+  let checkWin = false;
```

Comment:
This function would always return a truthy value ('You win' or 'You suck') so your parent function that checks for the return would always reset the stack. What would be better is if you just returned a truthy or falsy value and have the parent function return the message.